tools/runtests.sh

```diff
@@ -26,5 +26,5 @@ mkdir -p "$reports_dir"
 mkdir -p "$luacov_dir"
 
 cd "$base_dir"
-luarocks test --  "$@"
+luarocks test --local -- "$@"
```

Comment:
Without `--local` luarocks will install to the global directory where we don't have write permissions.